pages/dkp/kommander/2.3/clusters/custom-domain-certificate/configuration/index.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: layout.pug
+navigationTitle: Configuration 
+title: Configure a custom domain and certificate for your cluster
+menuWeight: 20
+excerpt: Configure a custom domain and certificate in the Management and any Managed or Attached Clusters
+beta: false
+enterprise: false
+---
+
+This section describes how to enable custom domains and certificates in your **Management and any Managed or Attached** clusters after the installation of DKP. If you want to set up a custom domain and certificate for your Management Cluster **during the installation**, refer to the [Customize a domain and certificate during installation][management] documentation.
+
+To customize the domain or certificate for a specific cluster after the installation of DKP, adapt or create an API `yaml` that allows DKP to implement the established adjustments on top of the default, or any previous configuration, you have made.
+
+## Configure custom domains or custom certificates
+
+<p class="message--warning"><strong>IMPORTANT:</strong> Ensure your <code>dkp</code> configuration references the management cluster of the environment where you want to customize the domain or certificate by setting the <code>KUBECONFIG=<path></code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
+
+To customize the domain or certificate of a cluster, alter the `spec` values of the `ingress` object in the `KommanderCluster`. Note that you can reference an issuer as an `issuerRef` **OR** a secret as a `certificateSecretRef`, as long as the object is created on the cluster where you want to customize the configuration.
+
+In the Management cluster, both the `KommanderCluster` and `issuerRef` or `certificateSecretRef` objects are on the same cluster. In Managed and Attached clusters, the `KommanderCluster` object is stored on the Management cluster, and the `issuerRef` or `certificateSecretRef` object is on the Managed or Attached cluster.
+
+Use the API yaml to customize the domain (via the `hostname` field), and the certificate (via the `issuerRef` or `certificateSecretRef` field).
+
+Refer to the following examples:
+
+1.  You have two options to create or update and apply the `KommanderCluster` object with the wanted ingress.
+
+    One option is to use a certificate that is managed automatically and supported by cert-manager like ACME (if you use Let's Encrypt, refer to the [example below](#example-configure-a-custom-certificate-with-lets-encrypt). For this, reference the `Issuer` or `ClusterIssuer` on the target cluster **to be used by cert-manager** in the `issuerRef` field, and enter the custom domain in the `hostname` field of the target cluster:
```

Comment:
@cneth this has changed significantly since the last time you reviewed. In the old version, we used `kubectl apply` to edit and apply the whole `KommanderCluster` resource yaml. In this version, we opted for using a patch command to keep users from accidentally editing other fields and also because it is a more straightforward way. 
In your review you suggested we add "Make sure the  `kubeconfigRef` field contains the name of the secret that holds the `kubeconfig` for the cluster." but I don't if that makes sense now that we don't use the whole yaml (however, it seems important!)
Maybe that is something we should elsewhere? maybe as a prereq? @floridoo @mhrabovcin